osf/migrations/__init__.py

```diff
@@ -135,4 +152,33 @@ def update_waffle_flags(sender, verbosity=0, **kwargs):
 
 def create_cache_table(sender, verbosity=0, **kwargs):
     if getattr(sender, 'label', None) == 'osf':
-        call_command('createcachetable', tablename=settings.CACHES[settings.STORAGE_USAGE_CACHE_NAME]['LOCATION'])
+        call_command('createcachetable', tablename=api_settings.CACHES[api_settings.STORAGE_USAGE_CACHE_NAME]['LOCATION'])
+
+
+def update_default_providers(sender, verbosity=0, **kwargs):
+    if getattr(sender, 'label', None) == 'osf':
+        if 'pytest' in sys.modules:
+            ensure_default_registration_provider()
```

Comment:
`else: ensure_default_providers` to maintain existing functionality to simplify local development

Reply:
Fixed to keep the original behavior. See the other comments on subjects.